plone/app/contenttypes/content.py

```diff
@@ -99,8 +99,9 @@ def PUT(self, REQUEST=None, RESPONSE=None):
     def get_size(self):
         return self.file.size
 
+    @property
```

Comment:
I prefer having a property here. This is a bit opinionated, but ``content_type`` reads like an attribute, whereas ``get_size`` reads like a method (because of the "get").

Opinions, @jensens @fgrcon ?